istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -58,12 +68,11 @@ spec:
 {{- if .Values.global.trustDomain }}
           - --trust-domain={{ .Values.global.trustDomain }}
 {{- end }}
+          - --keepaliveMaxServerConnectionAge
```

Comment:
What is this option ? Will make it impossible for the chart to work against 1.1 builds, can you remove ? Should have env variable or setting.

Reply:
https://github.com/istio/istio/blob/release-1.1/install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml#L77

Already in 1.1